storage/backendtester.go

```diff
@@ -60,8 +61,14 @@ func (b *BackendTest) NewBackendTest(t *testing.T) {
 }
 
 func (b *BackendTest) LocationTest(t *testing.T) {
-	if b.Backend.Location() != b.URL {
-		t.Errorf("%s: Expected %v, got %v", b.Description, b.URL, b.Backend.Location())
+	// since urls get parsed in BackendFromURL and some protocols use url-encoded parameters,
```

Comment:
Does this work without breaking the other tests?

Reply:
I think this should be fine... (until the tests will prove me wrong)